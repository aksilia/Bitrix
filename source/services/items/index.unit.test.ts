import Items from '.'

const mockCall = jest.fn(() => Promise.resolve()) as any
const mockList = jest.fn(() => Promise.resolve()) as any
const items = Items({ call: mockCall, list: mockList })

describe('Items', () => {
  beforeEach(() => {
    mockCall.mockClear()
    mockList.mockClear()
  })

  it('should expose API methods', () => {
    expect(items).toMatchSnapshot()
  })
})
