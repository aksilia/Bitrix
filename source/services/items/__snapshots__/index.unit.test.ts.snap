// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Items should expose API methods 1`] = `
Object {
  "create": [Function],
  "fields": [Function],
  "get": [Function],
  "list": [Function],
  "update": [Function],
}
`;
